src/getCoinCombination.test.js

```diff
@@ -7,4 +7,37 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should be non-negative input integer', () => {
+    expect(() => getCoinCombination(-1)).toThrow();
+  });
+
+  it(`should return array with length '4'`, () => {
+    expect(Array.isArray(getCoinCombination(1))).toBe(true);
+    expect(getCoinCombination(1)).toHaveLength(4);
+  });
+
+  it('should return [0, 0, 0, 0] for 0 cents', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it(`should return '[1, 0, 0, 0]' for 1 cents`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`should return '[1, 1, 0, 0]' for 6 cents`, () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
```

Comment:
The test description 'should return '[1, 1, 0, 0]' for 6 cents' is incorrect based on typical US coin denominations. The expected result should likely be '[1, 0, 1, 0]' for 6 cents, assuming the denominations are pennies, nickels, dimes, and quarters.

Reply:
i don't understand this issue. Re-phrase it on more time